examples/jenkins-integration/jenkinsfile

```diff
@@ -0,0 +1,175 @@
+pipeline {
+    agent any
+
+    environment {
+        def provision_successful = false
+        def vdb_id = ""
+        def vdb_name = ""
+        def DCT_HOSTNAME = "<INSERT>"
+        def SOURCE_VDB = "<INSERT>"
+        test_error = 'false'
+    }
+
+    stages {
+
+        stage ('Build Application') {
+            steps {
+                echo ('Building...')
+                sleep (50)
+            }
+        }
+        
+        stage ('Stand Up Full Application') {
+            parallel {
+                stage ('Apply Application Install') {
+                    steps{
+                        echo ('Provisioning Test App...')
+                        sleep (30)
+                    }
+                }
+                stage ('Create Database w/ Terraform') {
+                    steps {
+                        script {
+                            echo ('Provisioning  VDB...')
+                            git branch: 'main', changelog: false, poll: false, url: 'https://github.com/nick-mathison/delphix-terraform-examples.git'
```

Comment:
In a follow-up, we should update this to reference this folder. Not my personal one. I'll want to shut that one down.
We can handle that in a follow-up.

Reply:
I moved the simple-provision contents to the terraform repo and changed the reference here